stacks/DBStack.ts

```diff
@@ -14,6 +16,18 @@ export function DBStack({ stack, app }: StackContext) {
     primaryIndex: { partitionKey: 'counter' },
   });
 
+  const myTable = new dynamodb.Table(this, 'Table', {
+    partitionKey: { name: 'MyPartitionKey', type: dynamodb.AttributeType.STRING },
+    sortKey: { name: 'MySortKey', type: dynamodb.AttributeType.STRING },
+    removalPolicy: RemovalPolicy.DESTROY,
+  });
+  
+  myTable.addGlobalSecondaryIndex({
```

Comment:
Are you using the global secondary index ?

Reply:
you are right i am not using it. i deleted it in my last commit